stuff2sort/System.py

```diff
@@ -0,0 +1,1150 @@
+#!/usr/bin/env python
+
+#   // INITIAL DESCRIPTION //
+# XX AJ I don't know what to put here because the module only has this one class. What is the difference between the module description and the class description here?
+"""Short Module Description; Reference To Readme"""
+
+#   // MEATDATA //
+__author__ = 'Alina Jansen'
+__date__ = '2024-03-25'
+
+#   // IMPORTS //
+
+#   Imports Of Existing Libraries
+import re
+import os 
+import math
+import numpy as np
+
+#   Imports From Existing Libraries
+
+#   Imports Of Custom Libraries
+import Generators.GeneratorGeometries as geometry
+
+#   Imports From Custom Libraries
+from Logging.Logger import Logger
+from Generators import generate_pcf_from_top as make_pcf
+from Generators._helper import _flatten, get_linkatoms_ang
+
+#   // TODOS & NOTES //
+#   TODO:
+#   - Add the option to read atom input from a list (if that's what we want, wait for Florians opinion)
+#   - Add logger
+#   NOTE:
+
+#   // CLASS & METHOD DEFINITIONS //
+class SystemInfo():
+
+    '''
+    This Class Reads And Stores Information About The System
+    '''
+
+    def __init__(self, input_dict) -> None:
+
+        '''
+        ------------------------------ \\
+        EFFECT: \\
+        --------------- \\
+        Calls Functions To Read Information About The System\\
+        ------------------------------ \\
+        INPUT: \\
+        --------------- \\
+        input_dict: dict -> Dictionary Of The User Parameter Input \\
+        ------------------------------ \\
+        RETURN: \\
+        --------------- \\
+        NONE \\
+        ------------------------------ \\
+        '''
+
+        self.input_dict = input_dict
+
+        #   Make a list of all topology files
+        self.topology_list = self.getincludelist(self.input_dict['topologyfile'])
+
+        # Read The Different Types Of Molecules In The System
+        self.list_of_molecules = self.read_molecules()
```

Comment:
self.list_molecules is sufficient; leave out the 'of'.

Reply:
Done